docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -94,6 +94,20 @@ The following example uses implicit conversion to <xref:System.FormattableString
 
 :::code language="csharp" source="./snippets/string-interpolation.cs" id="Snippet4":::
 
+## Combining multiple interpolated strings
+
+To combine multiple interpolated strings, use the interpolation expression for each:
+
+```csharp
+var firstWord = "Hello";
+var secondWord = "World";
+
+var combinedInterpolatedString = $"{firstWord}, " + $"{secondWord}!";
```

Comment:
Why wouldn't this just be:

```
var combinedInterpolatedString = $"{firstWord}, {secondWord}!";
```

@BillWagner

Reply:
That would work, but wouldn't show the example as concatenating two interpolated strings. Admittedly, in this case, a single interpolated string would be clear. This would normally be done when all the source expressions were more complex.

Reply:
@gewarren it could be, for sure. I was using an example provided in the initial issue, which discussed concatenating multiple strings together specifically. So it may be that in the context of the example, the usefulness is lost.

Perhaps I could add something like "bear in mind, you could also accomplish this via" and use the syntax you've provided above?